### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by markkaylor**

### CODE REVIEW

File: starter/src/utils/localize.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  )
+
+  if (defaultLocaleIndex !== 0) {
+    // Grab the default locale from localaizedPaths array
+    const [defaultLoc] = localizedPaths.splice(defaultLocaleIndex, 1)
+    // Add it to the front
+    localizedPaths.unshift(defaultLoc)
+  }
+
+  return localizedPaths
```
Comment (markkaylor) [inline starter/src/utils/localize.js:64]: 🤷‍♂️

**Review by remidej — CHANGES_REQUESTED** [review]

File: starter/src/utils/localize.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export async function listLocalizedPaths(pageContext) {
+  const currentPage = {
+    locale: pageContext.locale,
+    href: localizePath(pageContext),
+  }
+
+  const paths = await Promise.all(
+    pageContext.localizations.map(async localization => {
+      const localePage = await fetchAPI(`/pages/${localization.id}`)
+      const page = { ...pageContext, ...localePage }
```
Comment (remidej) [inline starter/src/utils/localize.js:42]: I really like this

File: starter/src/utils/localize.js (hunk 1 above)
Comment (remidej) [reply starter/src/utils/localize.js:64]: Line 59 was a bit confusing to me, but it looks like it works

**Review by remidej — APPROVED** [review]
Everything looks good to me here too!

But just like for Next, let's wait for the template changes to be ready before we merge this

**Pull request merged**